diff -uNr a/opt/dassault-systemes/DraftSight/Resources/dassault-systemes_draftsight.desktop b/opt/dassault-systemes/DraftSight/Resources/dassault-systemes_draftsight.desktop
--- a/opt/dassault-systemes/DraftSight/Resources/dassault-systemes_draftsight.desktop	2014-07-03 22:33:32.000000000 +0400
+++ b/opt/dassault-systemes/DraftSight/Resources/dassault-systemes_draftsight.desktop	2014-07-03 22:38:27.000000000 +0400
@@ -1,11 +1,14 @@
 [Desktop Entry]
 Name=DraftSight
 Comment=Editing CAD images
-GenericName=DraftSight
-Exec="/opt/dassault-systemes/DraftSight/Linux/DraftSight"
+Comment[ru]=Редактирование изображений CAD
+GenericName=Professional CAD system
+GenericName[ru]=Профессиональная система CAD
+Exec=draftsight %u
 Path=/opt/dassault-systemes/DraftSight/Linux
 Type=Application
 Terminal=false
-Icon=dassault-systemes.draftsight
-MimeType=image/vnd.dwg;image/vnd.dwt;image/vnd.dxf;application-vnd.dassault-systemes.draftsight-dwg;application-vnd.dassault-systemes.draftsight-dwt;application-vnd.dassault-systemes.draftsight-dxf
-Categories=Application;Graphics;2DGraphics;RasterGraphics;
+Icon=dassault-systemes_draftsight
+Type=Application
+Categories=Application;Graphics;
+MimeType=application/dassault-systemes_draftsight;
diff -uNr a/opt/dassault-systemes/DraftSight/Resources/dassault-systemes_draftsight-dwg.xml b/opt/dassault-systemes/DraftSight/Resources/dassault-systemes_draftsight-dwg.xml
--- a/opt/dassault-systemes/DraftSight/Resources/dassault-systemes_draftsight-dwg.xml	2014-07-03 22:33:32.000000000 +0400
+++ b/opt/dassault-systemes/DraftSight/Resources/dassault-systemes_draftsight-dwg.xml	2014-07-03 22:22:58.000000000 +0400
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <mime-info xmlns='http://www.freedesktop.org/standards/shared-mime-info'>
-  <mime-type type="application/vnd.VAR_VENDOR_NAME.VAR_PRODUCT_NAME-dwg">
-    <sub-class-of type="image/vnd.dwg"/>
-    <comment>CAD Drawing</comment>
+  <mime-type type="application/dassault-systemes_draftsight">
+    <comment>CAD file</comment>
+    <glob pattern="*.dwg" priority="59"/>
+    <glob pattern="*.dxf" priority="59"/>
+    <glob pattern="*.dwt" priority="59"/>
     <magic priority="59">
       <match type="string" offset="0" value="AC10"/>
     </magic>
-    <glob pattern="*.dwg"/>
   </mime-type>
 </mime-info>
